components.md

```diff
@@ -4,13 +4,40 @@ title: Components
 permalink: /components/
 ---
 
-## Qt Auto Suite
+## Qt Automotive Suite
 
-[Qt Automotive Suite](https://www.qt.io/qt-automotive-suite/) is part of the Qt Project. There especially the [QtApplicationManager](https://doc.qt.io/QtApplicationManager/) and the [QtIvi](https://doc.qt.io/QtIVI/index.html) framework are relevant, as well as the [Neptune UI](https://doc.qt.io/NeptuneUI/index.html) which is an open source sample UI to get you started.
+[Qt Automotive Suite](https://www.qt.io/qt-automotive-suite/) is part of the Qt Project. It is a collection of software components and tools to enable development of In-Vehicle-Infotainment (IVI) systems. The vital components used in PELUX can be found below.
+
+### QtApplicationManager
+
+The [QtApplicationManager](https://doc.qt.io/QtApplicationManager/) provides the infrastructure for modern multi-process IVI architectures using Wayland composition and input event handling, while at the same time having the option to scale down to single-process setups for mid-range product variants, or for convenient native development on host platforms without Wayland. The Qt Application Manager supports you with app lifecylce management and isolating and securing apps.
+
+### QtIVI
+
+[QtIVI](https://doc.qt.io/QtIVI/) provides additional APIs commonly needed in automotive IVI or cluster systems. Since those APIs however differ slightly from project to project, a particular emphasis is put on easy extensibility/adaptability. QtIVI supports development by providing infrastructure for having multiple backends (e.g. simulation and the real hardware), as well as common diagnostic tooling.
```

Comment:
Shouldn't it say something about API between frontend (QML) and middleware services?

Reply:
Do we want to mention IVI Logging here too?

Reply:
I'm not sure any of the components in the baseline uses it, but I kind of think that we should mention it because it is a good component which brings good functionality and might be nice to know for the reader.

Reply:
@rpannek sure I can add that. 
You are right @jeremiah probably should have mentioned geniviextras (the DLT logging adapter).

Reply:
@rpannek the baseline doesn't use it really. Just the Qt Auto flavor.

Reply:
@GordanM FYI, "PELUX Baseline" also includes the Qt Auto variant of the platform.

Reply:
But QtAuto doesn't use IVI Logging?

Reply:
@rpannek IVI logging in this case would be DLT (geniviextras) that is what @jeremiah thought about, I presume.

Reply:
But IVI logging is not geniviextras, those are both projects which wrap DLT, geniviextras does it in a Qt context and IVI logging does it in a pure C++ context.

Reply:
@rpannek that's right, I am just thinking about what @jeremiah meant in this context.

Reply:
Actually, after this discussion, the situation is much clearer for me. I meant IVI logging but what I should have said was geniviextras.